R/tostring.R

```diff
@@ -367,6 +368,7 @@ new_line_warning <- function(str_v) {
 #' @return A string (`wrap_string` or character vector (`wrap_txt`) containing
 #'   the hard or soft word-wrapped content.
 #'
+#' @keyword internal
```

Comment:
Not sure what the value of not exporting this would be, really, but I suppose it might be possible (doesn't seem to be called directly by rtables in code or tests)

Reply:
I guess if users are not expecting to use this function, it should be internal, and not exported?

Reply:
That rule of thumb is often true generally, but isn't particulatly true  in a package like `formatters`. Most of what it exports is intended for _package developers_ who are using the framework (currently, this means the developers of `rtables` and `rlistings`). Many things are exported by `formatters` that should never be called by end users but cannot be internal to `formatters`.

`wrap_txt` in point of fact, is used in a function that is currently in `rlistings` (`vec_nlines`). That could b emigrated, and might even be better in `formatters`, but the payoff would not be worth the changes (even though they'd be fairly minimal) at this stage because `rlistings` is the only thing that needs it.

We could make `wrap_string` internal while keeping `wrap_txt` exported, but that would be pretty strange in my opinion, so I'm loathe to do that.